src/api/region.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+const API_BASE_URL = import.meta.env.VITE_API_BASE_URL;
+
+export const getRegion = async () => {
+  try {
+    const response = await axios.get(`${API_BASE_URL}/region`);
```

Comment:
I have changed the fetch, now is using axios. when using axios, we don't need to make the data into json with `const res = data.json()` because the Promise will be rejected.

using axios.get, we just need to return `response.data` and get the data array we wanted.